xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp

```diff
@@ -775,7 +775,11 @@ bool CPVRGUIInfo::GetListItemAndPlayerLabel(const CFileItem* item,
       case LISTITEM_EPISODE:
         if (epgTag->EpisodeNumber() >= 0)
         {
-          strValue = std::to_string(epgTag->EpisodeNumber());
+          if (epgTag->EpisodePart() > 0)
+            strValue =
+                StringUtils::Format("{0}/{1}", epgTag->EpisodeNumber(), epgTag->EpisodePart());
```

Comment:
Are we sure that in all languages the separator between episode number and episode part is a forward slash or should this rather be a translatable string making it possible to translate the separator?

Reply:
Fair question however if you can consider it fractional math I assumed the divisor was universal.  If you want the change I guess it would be a global language resource not skin.

Would this mean that the S and E need to be changed too?

Reply:
> Would this mean that the S and E need to be changed too?

Strictly speaking, yes. For example in German “episode” can be translated to “Episode”, but this is not a good translation. Best would be “Folge” but if we would use this for Kodi, due to the hard coded E “everywhere”, we would get inconsistent. Netflix for example uses “F” and “Folge”.  They do it right.